sil.cabal

```diff
@@ -55,11 +58,16 @@ library
                      , llvm-hs
                      , llvm-hs-pure
                      , text
+                     , megaparsec
+                     , strict
+                     , tasty
+                     , tasty-hunit
   extra-libraries:     gc
-                       jumper
+                     , jumper
+  extra-lib-dirs:      /home/hhefesto/src/stand-in-language/lib
```

Comment:
I'm not sure what to do about this. maybe commit a commented-out template for anyone to fill in if they want to use the repl.